content/markdown/concepts/emphasis/formating.md

```diff
@@ -50,3 +50,24 @@ For example:
 ```markdown
 This avacado egg sandwich is _very_ good.
 ```
+
+## Strikethrough 
+
+To make text strikethrough, use 2 ~ before and after a peace of text.
```

Comment:
```suggestion
To make text strikethrough, use 2 `~` before and after a piece of text.
```

When we include code in text, we'll wrap it in back ticks "`"

Reply:
In addition, the "~" is called a tilde, so we can write that out for clarity here ☺️ 
Another point is writing out numbers unless they're meant to be an integer data type, i.e. two instead of 2